tests/featureWriters/kernFeatureWriter_test.py

```diff
@@ -1496,74 +1490,55 @@ def test_kern_split_and_drop(FontClass, caplog):
 
     assert dedent(str(newFeatures)) == dedent(
         """\
-        @kern1.Grek.bar = [period];
-        @kern1.Grek.foo = [alpha];
-        @kern1.Latn.foo = [a];
-        @kern1.Orya.foo = [a-orya];
-        @kern2.Grek.bar = [period];
-        @kern2.Grek.foo = [alpha];
-        @kern2.Latn.foo = [a];
-        @kern2.Orya.foo = [a-orya];
-
-        lookup kern_Grek {
-            lookupflag IgnoreMarks;
-            pos @kern1.Grek.foo @kern2.Grek.bar 20;
-            pos @kern1.Grek.bar @kern2.Grek.foo 20;
-        } kern_Grek;
-
-        lookup kern_Latn {
+        @kern1.Cyrl_Grek_Latn_Orya.bar = [a-cy];
```

Comment:
I wonder if these class names may get too long, I believe FEA imposes some limits on the length. Maybe it's fine and we do already take care of ensuring the names are compliant and unique?